app/screens/ScheduleScreen/ScheduleDayPicker.tsx

```diff
@@ -71,31 +71,33 @@ export const ScheduleDayPicker: FC<ScheduleDayPickerProps> = ({
   const selectedScheduleIndex = scheduleDates.findIndex((date: Date) =>
     isSameDay(date, selectedScheduleDate),
   )
-  const containerRef = React.useRef<View>()
   const itemRefs = scheduleDates.map((_) => React.createRef<View>())
   const [measures, setMeasures] = React.useState<Measurement[]>(
     itemRefs.map((_) => {
       return { x: 0, y: 0, width: 0, height: 0 }
     }),
   )
 
-  const onLayout = React.useCallback(() => {
-    const m: Measurement[] = measures
-    itemRefs.forEach((itemRef, index) => {
-      itemRef.current?.measureLayout(
-        containerRef.current,
-        (x, y, width, height) => {
-          m[index] = { x, y, width, height }
-        },
-        () => {
-          m[index] = { x: 0, y: 0, width: 0, height: 0 }
-          reportCrash("ScheduleDayPicker-unable to measureLayout")
-        },
-      )
-    })
-
-    setMeasures(m)
-  }, [itemRefs])
+  const onLayout = useCallback(
+    ({ target }: { target: number }) => {
```

Comment:
update type to `LayoutChangeEvent` from `react-native`

Reply:
Addressed in [f54665e](https://github.com/infinitered/ChainReactApp2023/pull/212/commits/f54665e7affa49922804839adc04961b4f1bf9d2)